### ISSUE TITLE
Vanilla_U-Net_Example_on_RS.html - adding comments and descriptions

### ISSUE THREAD

**Issue by Willfriedrichs** [CPLN-680-Spring-2022/Tan_Jiamin_RoadDetection#3]
file:///C:/Users/willf/Desktop/Tan_Jiamin_RoadDetection-main/Vanilla_U-Net_Example_on_RS.html

I was a little confused by this file. The note was helpful, I understand that this is an example of U-Net implementation on images to produce a training mask and make a prediction. Is there a specific reason why this file is not in “codes”? A little introduction at the top of the document would do a lot to help make this more understandable for someone looking at it for the first time. 

It would be helpful to my understanding of the code if there were more comments throughout. I see code that is commented out, but I for the time being, I am not seeing any comments explaining what each chunk of code is doing. Maybe adding sections could also help with the organization, and with making it more easily understandable for others (or for yourself in the future!). 

In terms of “code smell”, I see some areas here where there is some repetition in the structure of the code. For example, many times in arow, an object is created with functions from tf.keras.layers. it looks to me like these functions are repeated in the same order several times. This part of the code could be a good candidate for being changed to a method that is run several times with different inputs. Such a change would reduce the amount of lines, and make the code more easily readable and understandable.
